### ISSUE TITLE
Add a accuracy calculate function 

### ISSUE THREAD

**Issue by M0hammadL** [theislab/multimil_reproducibility#2]
we need a helper function to calculate the  the metrics using https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html receiving data
